src/compiler/error-detector.js

```diff
@@ -8,6 +8,8 @@ const prohibitedKeywordRE = new RegExp('\\b' + (
   'super,throw,while,yield,delete,export,import,return,switch,default,' +
   'extends,finally,continue,debugger,function,arguments'
 ).split(',').join('\\b|\\b') + '\\b')
+const deleteExpressionRE = /\bdelete\s*\([^\)]*\)/
```

Comment:
In the `with` statement call the `delete` method, always use the call is the global delete, which is a BUG need to compile at compile time